transport/grpc/options.go

```diff
@@ -119,25 +121,35 @@ func ClientMaxSendMsgSize(clientMaxSendMsgSize int) TransportOption {
 	}
 }
 
-// ClientTLS says to use TLS with the system certificate pool on the client side.
-//
-// The default is to not use TLS.
-func ClientTLS() TransportOption {
-	return func(transportOptions *transportOptions) {
-		transportOptions.clientTLS = true
-	}
-}
-
 // InboundOption is an option for an inbound.
 type InboundOption func(*inboundOptions)
 
 func (InboundOption) grpcOption() {}
 
+// Creds returns an InboundOption that sets credentials for incoming
+// connections.
+func Creds(creds credentials.TransportCredentials) InboundOption {
```

Comment:
If possible, I'd like to avoid this name. Creds and WithTransportCredentials are not very symmetric. Plus we don't use "With" in our existing options.

Can we call these, `InboundCredentials` and `DialerCredentials`?

Reply:
These are exactly the assymetric names used upstream by grpc. I didn't really like their naming either. I am OK changing them even though that diverges from the options that we are wrapping here.

Reply:
Yeah, let's diverge from them to match the naming conventions in the rest of YARPC. InboundCredentials and DialerCredentials is a lot more obvious about what it's for and which direction it applies in.

Reply:
Done.